neuraxle/base.py

```diff
@@ -1064,19 +1573,21 @@ def _load_saved_pipeline_steps_before_index(self, saved_pipeline: 'TruncableStep
 
     def _set_steps(self, steps_as_tuple: NamedTupleList):
         """
-        Set pipeline steps
+        Set steps as tuple.
```

Comment:
I know it's not really the goal of this PR, but I think this method should be public just like the `MetaStepMixin.set_step`, here the method is just to the plural and ends with an "s".